test/integration/features/proxy.feature

```diff
@@ -0,0 +1,87 @@
+@proxy @minishift-only
+Feature: Minishift can run behind proxy
+  As a user I can use Minishift behind a proxy.
+
```

Comment:
No, that was not the case. So I am completely unsure how is it possible that it asked you for a password.

+1 on keeping test setup information in feature header, seems practical. Will add it.